### DISCUSSION TITLE
Reduce confirmation dialogs for deleting on device files

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#947] [Enhancements]
Issue:

On Android, while deleting files which are not backed up, the app shows three dialogs
1) Normal confirmation dialog (ente)
2) Warning dialog about permanent deletion (ente)
3) System confirmation dialog

Roadmap: https://roadmap.ente.io/faster-delete-feature-p-8218/

**Comment by XieyilinMain** [comment]
Me too. The confirmation step should be happened right after pressing delete key.
Then there would be only 2 steps.

**Reply by XieyilinMain** [reply]
@ashilkn
